src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -878,7 +878,7 @@ protected function chartBuildSettings(array $records) {
     $tick_values_field = $this->config('axis', 'x', 'tick', 'values', 'field');
 
     if ($settings['axis']['x']['type'] === '' && !empty($record) && property_exists($record, $tick_values_field)) {
-      is_numeric($record->{$tick_values_field}) ?
+      is_numeric($record->{$tick_values_field} || $record[0] == $record->{$tick_values_field}) ?
```

Comment:
A few things here @WilsonWu2100:
- Is the tick values field expected to be `"0"`? If so then this is too specific
- `$record` is an object: https://github.com/govCMS/dvf/blob/2.x/dvf_csv/src/Plugin/Visualisation/Source/CsvFile.php#L182
so I am surprised that you do not get an error when doing $record[0] because I do...

Regarding the issue itself, I could not reproduce it following the steps you have highlighted in the story. If the "tick_values_field" is "Fruits" then `is_numeric()` will be false here so don't really understand the issue.

cc @jez500

Reply:
@jez500 I've made the changes to the fix, could you please have a review?
@thery-dh If you follow the testing steps in the ticket, I think you should be able to see the issues in the screenshot attached to that ticket.